src/client/mod.rs

```diff
@@ -97,6 +100,61 @@ impl GaloyClient {
         Ok(me)
     }
 
+    pub fn fetch_balance(
+        &self,
+        wallet: Option<Wallet>,
```

Comment:
why is the `wallet` parameter there?
shouldn't it rather be `walletType`?
and where is it used?

Reply:
We have the option to specify `--btc` or `--usd` along with the wallet_ids. So here `wallet` is used when one of these options is given as input.
Renamed it to `wallet_type` based on your feedback. 626873f802763ef53fbef0cf9b53c8a89a3fe5c5